.github/workflows/sub-ci-unit-tests-docker.yml

```diff
@@ -53,10 +53,15 @@ jobs:
       - name: Run all tests
         run: |
           docker pull ${{ vars.GAR_BASE }}/${{ vars.CI_IMAGE_NAME }}@${{ inputs.image_digest }}
-          docker run -t \
+          docker run --tty \
           -e RUN_ALL_TESTS=1 \
           -e FEATURES="journald prometheus filter-reload" \
           -e NETWORK="${{ inputs.network || vars.ZCASH_NETWORK }}" \
+          -e RUST_LOG=${{ env.RUST_LOG }} \
+          -e RUST_BACKTRACE=${{ env.RUST_BACKTRACE }} \
+          -e RUST_LIB_BACKTRACE=${{ env.RUST_LIB_BACKTRACE }} \
+          -e COLORBT_SHOW_HIDDEN=${{ env.COLORBT_SHOW_HIDDEN }} \
```

Comment:
This variable should configure [`color-backtrace`](https://github.com/athre0z/color-backtrace), but I didn't find any reference to it in the codebase, so it looks like we're not using it anymore. It might be worth removing the variable altogether.